apps/nrm/translations/src/en/fields.json

```diff
@@ -547,17 +547,19 @@
     "paragraph-1": "If a potential victim wants you to refer their case, they can get:",
     "bullet-1": "access to support (for example, housing and legal advice)",
     "bullet-2": "a formal decision – this can help if they’re being prosecuted for a crime they were forced to commit",
-    "div-1-paragraph-1": "We will need to share their personal details with organisations such as the police and the Home Office if they agree to their case being referred.",
-    "div-1-paragraph-2": "Please explain to the potential victim the ways in which the Home Office may process their personal information, including the purposes for which it will be used, the legal basis and who it will be shared with. This is contained in the <a class=\"govuk-link\" href=\"https://www.gov.uk/government/publications/human-trafficking-victims-referral-and-assessment-forms/privacy-information-notice-national-referral-mechanism\">Privacy Information Notice</a> and <a class=\"govuk-link\" href=\"https://www.gov.uk/government/organisations/home-office/about/personal-information-charter\">Personal Information Charter.</a>",
+    "nrm-checks-p": "You should:",
+    "div-1-paragraph-1": "make sure that the potential victim understands by entering the NRM they are consenting to a reasonable grounds and conclusive grounds decision being made",
+    "div-1-paragraph-2": "make sure that the potential victim understands the support that they may be able to receive, as detailed in the <a class=\"govuk-link\" href=\"https://assets.publishing.service.gov.uk/media/67812b3b00e3d719f19217eb/Modern+Slavery+Statutory+Guidance+_EW_+and+Non-Statutory+Guidance+_SNI_+v3.12.pdf\">guidance (see section 'Support for Adult Victims').</a> This may require the assistance of an appropriate interpreter.",
```

Comment:
The business wanted the pdf link to go to the exact page in the pdf. You can do this by adding `#page=65` after `.pdf` in the link